src/directory/directory.js

```diff
@@ -6,13 +6,18 @@ const directory = {
     },
     items: {
       devpreview: {
-        title: "New! Amplify iOS (Developer Preview)",
+        title: "New! Amplify (Developer Preview)",
         items: [
           {
             title: "Getting started",
             route: "/lib/devpreview/getting-started",
             filters: ["ios"],
           },
+          {
+            title: "Getting started",
+            route: "/lib/devpreview/getting-started",
+            filters: ["android"],
+          },
```

Comment:
This extra "Getting started" block can be deleted and "android" can be added to the `filters` list on line 14.